nginx.conf

```diff
@@ -52,14 +52,36 @@ http {
       # allow for HTML "meta redirects" on pages of sites that have been migrated
       # to Federalist. For context, see https://github.com/18F/federalist-proxy/issues/19.
       #
-      # More extensions can be added inside the parentheses, separated 
+      # More extensions can be added inside the parentheses, separated
       # by | characters, e.g. `\.(cfm|php)$`.
       location ~* \.(cfm)$ {
         add_header Content-Type "text/html";
         # Notice there is no trailing slash on the proxy_pass url here.
         proxy_pass <%= ENV["FEDERALIST_S3_BUCKET_URL"] %>;
       }
 
+      # Enable CORS support.
+      # Based on https://enable-cors.org/server_nginx.html
+      set $cors_origins 'https://s.codepen.io';
```

Comment:
Presumably you'll want to set this to `*`.

Reply:
Unfortunately, this is intentional

Reply:
☝️